contracts/staking/StakingVerifier.sol

```diff
@@ -279,36 +312,38 @@ contract StakingVerifier {
     }
 
     /// @dev Changes staking parameter limits.
-    /// @param _rewardsPerSecondLimit Rewards per second limit.
+    /// @param _minStakingDepositLimit Minimum staking deposit limit.
     /// @param _timeForEmissionsLimit Time for emissions limit.
     /// @param _numServicesLimit Limit for the number of services.
+    /// @param _apyLimit APY limit in 1e18 format.
     function changeStakingLimits(
-        uint256 _rewardsPerSecondLimit,
+        uint256 _minStakingDepositLimit,
         uint256 _timeForEmissionsLimit,
-        uint256 _numServicesLimit
+        uint256 _numServicesLimit,
+        uint256 _apyLimit
     ) external {
         // Check the contract ownership
         if (owner != msg.sender) {
             revert OwnerOnly(owner, msg.sender);
         }
 
         // Zero values check
-        if (_rewardsPerSecondLimit == 0 || _timeForEmissionsLimit == 0 || _numServicesLimit == 0) {
+        if (_minStakingDepositLimit == 0 || _timeForEmissionsLimit == 0 || _numServicesLimit == 0 || _apyLimit == 0) {
             revert ZeroValue();
         }
 
```

Comment:
where's the setter for _minStakingDepositLimit?

Reply:
Constructor and `changeStakingLimits()` function, it just replaces `rewardsPerSecondLimit` everywhere.

Reply:
Yes but see below,l we are not able to update _minStakingDepositLimit as the setter is missing